<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
    <ros2_control name="ContinuumRobotSensor" type="sensor">
        <hardware>
            <plugin>cr_sensor_interface/CrSensorInterface</plugin>
            <param name="robot1_load_cell1_serialNum">566303</param>
            <param name="robot1_load_cell2_serialNum">587030</param>
            <param name="robot2_load_cell1_serialNum">587030</param>
            <param name="robot2_load_cell2_serialNum">580129</param>
        </hardware>

       <sensor name="voltageRatios_sensor1">
        <state_interface name="voltageRatio10"/>
        <state_interface name="voltageRatio11"/>
        <state_interface name="voltageRatio12"/>
        <state_interface name="voltageRatio13"/>
        <param name="frame_id">tool_link</param>
        <param name="voltageRatio10_range">1000</param>
        <param name="voltageRatio11_range">1000</param>
        <param name="voltageRatio12_range">1000</param>
        <param name="voltageRatio13_range">1000</param>
      </sensor>
       <sensor name="voltageRatios_sensor2">
        <state_interface name="voltageRatio20"/>
        <state_interface name="voltageRatio21"/>
        <state_interface name="voltageRatio22"/>
        <state_interface name="voltageRatio23"/>
        <param name="frame_id">tool_link</param>
        <param name="voltageRatio20_range">1000</param>
        <param name="voltageRatio21_range">1000</param>
        <param name="voltageRatio22_range">1000</param>
        <param name="voltageRatio23_range">1000</param>
      </sensor>
    </ros2_control>
</robot>
